node_modules/oae-core/editprofile/js/editprofile.js

```diff
@@ -111,9 +111,29 @@ define(['jquery', 'oae.core'], function ($, oae) {
             }
 
             // Render the form elements
+            // detect if is possible to modify the name and email
+            var disableModification = false;
+            var EnableModification = true;
+            var authenticationStrategy = oae.data.me.authenticationStrategy;
+           if ((authenticationStrategy === 'cas')| (authenticationStrategy === 'shibboleth')){
+            if (oae.api.config.getValue('oae-authentication', 'shibboleth', 'enabled')){
+                if (oae.api.config.getValue('oae-authentication', 'shibboleth', 'disableProfile')){
```

Comment:
And this could be `if (oae.api.config.getValue('oae-authentication', 'shibboleth', 'disableProfile') || oae.api.config.getValue('oae-authentication', 'cas', 'disableProfile'))`.